### DISCUSSION TITLE
How to automatically introduce classes defined in the utils folder?

### DISCUSSION THREAD

**Discussion by lxccc812** [nuxt/nuxt#26620] [Questions] [answered]
Functions created under the utils folder can be automatically introduced by other files without manual introduction. So if a class is declared under utils, is there a way to automatically introduce it?

**Comment by xjccc** [comment]
Have a try ~

```utils/index.ts
export class A {
  constructor (public a: string) {
      this.a = a
  }
}

```

```app.vue
<script setup lang=ts>
const a = new A('1')
</script>

```

**Reply by lxccc812** [reply]
Name 'A' not found. Did you mean "a"?

This does not find class A
![image](https://github.com/nuxt/nuxt/assets/110974969/acc350a0-5990-47a6-9dce-51455b6b0094)

**Reply by xjccc** [reply]
What is your nuxt version?I'm use the latest.

https://stackblitz.com/edit/nuxt-starter-mpz8hk?file=utils%2Ftest.ts,app.vue

**Reply by lxccc812** [reply]
nuxt@3.9.2
But after I run `pnpm update`, after updating these dependencies.
![image](https://github.com/nuxt/nuxt/assets/110974969/3d22bbb8-3d5d-4a30-876f-a3fe69eac5c6)
The error disappeared. I think there is no error in the example, but there is a problem with the format check.

**Reply by xjccc** [reply]
maybe, you should `pnpm dev`

**Reply by lxccc812** [reply]
I think I found the answer to my question, I should use `export class A` instead of `export default class A`

**Reply by xjccc** [reply]
ok..i see your picture show `export`

**Reply by lxccc812** [reply, marked as answer]
My local uses `export default`
Check out this [link](https://stackblitz.com/edit/nuxt-starter-msoda1?file=pages%2Ftest1.vue,utils%2Ftest1.ts,pages%2Ftest2.vue,utils%2Ftest2.ts,pages%2Ftest3.vue,utils%2Ftest3.ts)

Also, if you want to use `export default`, then you should instantiate the file name instead of the defined class name.
